models/articles_models.js

```diff
@@ -10,3 +12,21 @@ exports.selectArticleById = (id) => {
       return result.rows[0];
     });
 };
+
+exports.updateArticleById = (article_id, votes) => {
+  if (votes && typeof votes !== "number") {
+    return Promise.reject({ status: 400, msg: "Invalid input" });

```

Comment:
here you don't need to reject if typeof votes is not "number", if a string goes into the query it will be rejected automatically because it will break

Reply:
and then we can handle the sql error which is returned